tests/flow/test_traversal_construction.py

```diff
@@ -0,0 +1,75 @@
+import os
+from RLTest import Env
+from redisgraph import Graph
+
+graph = None
+
+class testTraversalConstruction():
```

Comment:
These tests seem considerably less exhaustive to me than what this PR had attempted previously. At minimum, it should test different combinations of labeled and filtered entities; right now there is not even a test to validate that filtered entities are preferred over labeled entities.